pom.xml

```diff
@@ -67,7 +67,7 @@
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.319.1</jenkins.version>
+    <jenkins.version>2.334</jenkins.version>
```

Comment:
is there a reason for 2.334?

as far as I can tell the tables have been available since 2.321 which means they are included in the:
2.332.1 LTS?

Doesn't matter too much since https://github.com/jenkinsci/credentials-plugin/pull/287 needs to bump to 2.340